src/main.py

```diff
@@ -87,8 +87,8 @@
             pass
         case 2:
             # one argument
-            print("Feature not implemented yet.", file=sys.stderr)
-            sys.exit(1)
+            print(translate_dna(open(infile, "r")), file=sys.stderr) #("Feature not implemented yet.", file=sys.stderr)
```

Comment:
If you open a file here, you still want to run the code below. So you shouldn't call `translate_dna()` but just open the file.

```python
infile = open(argv[1], 'r')
```